src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -103,6 +103,27 @@ Copyright (c) .NET Foundation. All rights reserved.
     <GenerateResourceMSBuildRuntime>CurrentRuntime</GenerateResourceMSBuildRuntime>
   </PropertyGroup>
 
+  <!-- We need to determine if the RuntimeIdentifier was globally set. This check takes advantage of the fact that
+       global properties cannot be overwritten. -->
+  <PropertyGroup>
+    <!-- First, backup the value of the RuntimeIdentifier -->
+    <GlobalRuntimeIdentifier>$(RuntimeIdentifier)</GlobalRuntimeIdentifier>
+    <!-- Now set the RuntimeIdentifier to a bogus value. If it is global, this change will be ignored. -->
+    <RuntimeIdentifier>__________</RuntimeIdentifier>
+  </PropertyGroup>
+
+  <!-- If the RuntimeIdentifier was not set globally, put back its local value. -->
+  <PropertyGroup Condition=" '$(RuntimeIdentifier)' != '$(GlobalRuntimeIdentifier)'">
+    <RuntimeIdentifier>$(GlobalRuntimeIdentifier)</RuntimeIdentifier>
+    <!-- Null out the GlobalRuntimeIdentifier since RuntimeIdentifier is not set globally. -->
+    <GlobalRuntimeIdentifier></GlobalRuntimeIdentifier>
+  </PropertyGroup>
+
+  <!-- If the RuntimeIdentifier was set globally, pass it to GetTargetFrameworkProperties. -->
+  <PropertyGroup Condition=" '$(GlobalRuntimeIdentifier)' != '' ">
+    <AdditionalPropertiesForGetTargetFrameworkProperties>;ReferringRuntimeIdentifier=$(GlobalRuntimeIdentifier)</AdditionalPropertiesForGetTargetFrameworkProperties>
```

Comment:
Where is `AdditionalPropertiesForGetTargetFrameworkProperties` consumed? I don't see it in the MSBuild side of this PR or here.